Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -411,8 +412,7 @@ private async Task SendInviteEmail(int teamID, string inviteeUsername, string in
                  $"Hey {invitee.FirstName}!<br><br>" +
                 $"{inviter.FirstName} {inviter.LastName} has invited you join <b>{team.Name}</b> for <b>{team.Activity.Name}</b> <br>" +
                 $"Registration closes on <i>{team.Activity.RegistrationEnd.ToString("D", CultureInfo.GetCultureInfo("en-US"))}</i> <br>" +
-                //$"check it out <a href='https://360.gordon.edu/recim'>here</a>! <br><br>" + //for production
-                $"check it out <a href='https://360recim.gordon.edu/recim'>here</a>! <br><br>" +//for development
+                $"check it out <a href='{_config["RecIM_Url"]}'>here</a>! <br><br>" +
```

Comment:
"Click Here" links are generally frowned upon as they seem like they're trying too hard and people tend to think they are either spam or phishing. Perhaps this would be better wording?
```suggestion
                $"<a href='{_config["RecIM_Url"]}'>check it out on 360</a>!<br><br>" +
```